KerbalStuff/blueprints/mods.py

```diff
@@ -376,21 +375,15 @@ def delete(mod_id: int) -> werkzeug.wrappers.Response:
             editable = True
     if not editable:
         abort(403)
-    for featured in Featured.query.filter(Featured.mod_id == mod.id).all():
-        db.delete(featured)
-    for media in Media.query.filter(Media.mod_id == mod.id).all():
-        db.delete(media)
-    for referral in ReferralEvent.query.filter(ReferralEvent.mod_id == mod.id).all():
-        db.delete(referral)
-    for version in ModVersion.query.filter(ModVersion.mod_id == mod.id).all():
-        db.delete(version)
-    db.delete(mod)
-    db.commit()
-    notify_ckan(mod, 'delete', True)
+
     storage = _cfg('storage')
     if storage:
         full_path = os.path.join(storage, mod.base_path())
-        rmtree(full_path)
+        rmtree(full_path, ignore_errors=True)
```

Comment:
If a mod has been renamed, then `mod.base_path()` won't necessarily contain all the download files. We probably need to `os.remove` each `ModVersion.download_path`.